kubernetes/uptime-kuma/ingressroute.yml

```diff
@@ -0,0 +1,39 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    kubernetes.io/ingress.class: nginx
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "3600"
+    nginx.ingress.kubernetes.io/proxy-send-timeout: "3600"
+    nginx.ingress.kubernetes.io/server-snippets: |
+      location / {
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_http_version 1.1;
+        proxy_set_header X-Forwarded-Host $http_host;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header X-Forwarded-For $remote_addr;
+        proxy_set_header Host $host;
+        proxy_set_header Connection "upgrade";
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header   Upgrade $http_upgrade;
+        proxy_cache_bypass $http_upgrade;
+        }
+  name: uptime-kuma-ingress
+spec:
+  tls:
+  - hosts:
+    - monitor.cxde.link
+    secretName: monitor-cxde-link-tls
+  rules:
+  - host: monitor.cxde.link
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: uptime-kuma-uptime-kuma
```

Comment:
service name should be `uptime-kuma`, here that name is doubled?

Reply:
doubled because of the namespace, i will try it without the second one just in a moment

Reply:
i edited the files and changed a little thing in the README of the kubernetes Folder.
And again, ready for review @louislam @srgvg